ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift

```diff
@@ -87,36 +89,65 @@ class DateTimePickerDemoController: DemoController {
         dateTimePicker.present(from: self, with: .dateTime, startDate: startDate ?? Date(), datePickerType: datePickerType)
     }
 
+    func calcDatePickerParams() -> (startDate: Date, endDate: Date, titles: DateTimePicker.Titles) {
+        let startDate = startDate ?? Date()
+        let endDate = endDate ?? Calendar.current.date(byAdding: .day, value: 1, to: startDate) ?? startDate
+        let titles: DateTimePicker.Titles
+        if datePickerType == .calendar && !UIAccessibility.isVoiceOverRunning {
+            titles = .with(startSubtitle: "Assignment Date", endSubtitle: "Due Date")
+        } else {
+            titles = .with(startTab: "Assignment Date", endTab: "Due Date")
+        }
+        return (startDate, endDate, titles)
+    }
+
     @objc func presentDateRangePicker() {
-        let startDate = self.startDate ?? Date()
-        let endDate = self.endDate ?? Calendar.current.date(byAdding: .day, value: 1, to: startDate) ?? startDate
+        let (startDate, endDate, _) = calcDatePickerParams()
         dateTimePicker.present(from: self, with: .dateRange, startDate: startDate, endDate: endDate, datePickerType: datePickerType)
     }
 
     @objc func presentTabbedDateRangePicker() {
-        let startDate = self.startDate ?? Date()
-        let endDate = self.endDate ?? Calendar.current.date(byAdding: .day, value: 1, to: startDate) ?? startDate
+        let (startDate, endDate, _) = calcDatePickerParams()
         dateTimePicker.present(from: self, with: .dateRange, startDate: startDate, endDate: endDate, datePickerType: datePickerType, dateRangePresentation: .tabbed)
     }
 
     @objc func presentDateTimeRangePicker() {
-        let startDate = self.startDate ?? Date()
-        let endDate = self.endDate ?? Calendar.current.date(byAdding: .hour, value: 1, to: startDate) ?? startDate
+        let (startDate, endDate, _) = calcDatePickerParams()
         dateTimePicker.present(from: self, with: .dateTimeRange, startDate: startDate, endDate: endDate, datePickerType: datePickerType)
     }
 
     @objc func presentCustomSubtitlePicker() {
-        let startDate = self.startDate ?? Date()
-        let endDate = self.endDate ?? Calendar.current.date(byAdding: .day, value: 1, to: startDate) ?? startDate
-        let titles: DateTimePicker.Titles
-        if datePickerType == .calendar && !UIAccessibility.isVoiceOverRunning {
-            titles = .with(startSubtitle: "Assignment Date", endSubtitle: "Due Date")
-        } else {
-            titles = .with(startTab: "Assignment Date", endTab: "Due Date")
-        }
+        let (startDate, endDate, titles) = calcDatePickerParams()
         dateTimePicker.present(from: self, with: .dateRange, startDate: startDate, endDate: endDate, datePickerType: datePickerType, titles: titles)
     }
 
+    @objc func presentLeftGlyphPicker() {
```

Comment:
glyph isn't really Apple HIG terminology.

Reply:
Removed the term 'glyph' from all code and also from Demo project

Reply:
Removed all references to 'glyph'